EduVPN/Controllers/SearchViewController.swift

```diff
@@ -77,17 +83,45 @@ final class SearchViewController: ViewController, ParametrizedViewController {
         let persistenceService = parameters.environment.persistenceService
         persistenceService.hasServersDelegate = self
         hasAddedServers = persistenceService.hasServers
+
+        if parameters.shouldAutoFocusSearchField {
+            showTableView(animated: false)
+        }
+    }
+
+    #if os(macOS)
+    override func viewDidAppear() {
+        if parameters.shouldAutoFocusSearchField {
+            self.view.window?.makeFirstResponder(searchField)
+        }
```

Comment:
You may want to clear shouldAutoFocusSearchField so this only happens on the first appear event.

Reply:
Good catch, thanks. 👍